.goreleaser.yml

```diff
@@ -133,6 +133,13 @@ builds:
       - -trimpath
       - -s -w
 
+release:
+  github:
+    owner: labring
+    repo: sealos
+  draft: true
```

Comment:
If set to true, will not auto-publish the release.

draft Is this setting inappropriate?

Reply:
Yes, for testing purposes. After the release workflow is stable, I'll change `draft` to `false`.